### PR TITLE
DEVDOCS-4400: [net new] Tax Rates Zones

### PR DESCRIPTION
**Pull request by bc-andreadao**
# [DEVDOCS-4400](https://bigcommercecloud.atlassian.net/browse/DEVDOCS-4400)

This PR is related to:
-[PR 945](https://github.com/bigcommerce/api-specs/pull/945): API Spec
-[PR 133](https://github.com/bigcommerce-labs/next-dev-center-prototype/pull/133): redirects

### CODE REVIEW

**Comment by bc-andreadao** [comment]
Planning on testing the example requests when the endpoints are ready

File: docs/api-docs/tax/tax-rates-zones.md [hunk 1]
```diff
@@ -0,0 +1,464 @@
+# Tax Rates and Tax Zones
+
+[Tax rates](https://support.bigcommerce.com/s/article/Manual-Tax-Setup?language=en_US#tax-rate) are percentages for which you tax products or services. Tax zones are geographic locations that you define in your store. [Tax zones](https://support.bigcommerce.com/s/article/Manual-Tax-Setup?language=en_US#tax-zone) allow you to vary tax rates for different customer groups and locations. You can account for multiple tax jurisdictions by adding multiple tax rates to a zone. Tax rates and zones apply for all storefronts on a store.
```
Comment (theromulans) [inline docs/api-docs/tax/tax-rates-zones.md:3]: > You can account for multiple tax jurisdictions by adding multiple tax rates to a zone

I'm not sure about this, I'd probably drop the sentence.

Multiple jurisdictions = multiple tax zones. Multiple tax rates only speaks to use cases such as compounding tax rates and offering a further, detailed breakdown.

File: docs/api-docs/tax/tax-rates-zones.md [hunk 2]
```diff
@@ -0,0 +1,464 @@
+# Tax Rates and Tax Zones
+
+[Tax rates](https://support.bigcommerce.com/s/article/Manual-Tax-Setup?language=en_US#tax-rate) are percentages for which you tax products or services. Tax zones are geographic locations that you define in your store. [Tax zones](https://support.bigcommerce.com/s/article/Manual-Tax-Setup?language=en_US#tax-zone) allow you to vary tax rates for different customer groups and locations. Tax rates and zones apply for all storefronts on a store.
```
Comment (bc-traciporter) [inline docs/api-docs/tax/tax-rates-zones.md:3]:
```suggestion
[Tax rates](https://support.bigcommerce.com/s/article/Manual-Tax-Setup?language=en_US#tax-rate) are percentages for which you tax products or services. Tax zones are geographic locations that you define in your store. [Tax zones](https://support.bigcommerce.com/s/article/Manual-Tax-Setup?language=en_US#tax-zone) allow you to vary tax rates for different customer groups and locations. Tax rates and zones apply for all storefronts in a store.
```

File: docs/api-docs/tax/tax-rates-zones.md [hunk 3]
```diff
@@ -0,0 +1,464 @@
...
+
+This guide shows you how to use the Tax Rates and Tax Zones API. The Tax Rates and Zones API allows you to configure [manual taxes](https://support.bigcommerce.com/s/article/Manual-Tax-Setup?language=en_US) in a store. For more info, see the [Tax Rates and Tax Zones API Reference](/api-reference/store-management/tax-rates-and-zones).
```
Comment (bc-traciporter) [inline docs/api-docs/tax/tax-rates-zones.md:5]:
```suggestion
This guide shows you how to use the Tax Rates and Tax Zones API. The Tax Rates and Zones API allows you to configure [manual taxes](https://support.bigcommerce.com/s/article/Manual-Tax-Setup?language=en_US) in a store. For more information, see the [Tax Rates and Tax Zones API Reference](/api-reference/store-management/tax-rates-and-zones).
```

**Review by bc-traciporter — COMMENTED** [review]
Made editorial suggestions

**Review by markcmurphy — APPROVED** [review]

**Comment by bc-andreadao** [comment]
@slsriehl Okay, I looked back at my notes and realized I mispoke in the slack conversation [here](https://bigcommerce.slack.com/archives/C039JSWHH50/p1671612484495279?thread_ts=1671578023.673549&cid=C039JSWHH50). The original intention of [line 15](https://github.com/bigcommerce/dev-docs/blob/e6c9f4061b76d0dacbe2fa607bb55e9efdb8e4c8/docs/api-docs/tax/tax-rates-zones.md?plain=1#L15) was to talk about price display settings for taxes (not the prices for each product in Price Lists). I changed Line 15 to reflect this. 

Good to merge this doc now

**Review by slsriehl — APPROVED** [review]
🎉🎉🎉

**Pull request merged**